modules/ossm-security-cert-manage.adoc

```diff
@@ -0,0 +1,90 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/service_mesh_user_guide/ossm-security.adoc
+
+[id="ossm-cert-manage"]
+= Adding an external certificate authority key and certificate
+
+By default, {ProductName} generates self-signed root certificate and key, and uses them to sign the workload certificates. You can also use the operator-specified certificate and key to sign workload certificates, with operator-specified root certificate. This task demonstrates an example to plug certificates and key into {ProductShortName}.
+
+
+
+.Prerequisite
+
+* You must have installed {ProductName} with mutual TLS enabled to configure certificates.
+* Deploy the Bookinfo sample application to verify the results.
+
+== Adding an existing certificate and key
+
+To us an existing signing (CA) certificate and key, you must create a chain of trust file that includes the CA certificate, key, and root certificate. You must use the following exact file names for each of the corresponding certificates. The CA certificate is called `ca-cert.pem`, the key is `ca-key.pem`, and the root certificate, which signs `ca-cert.pem`, is called `root-cert.pem`. If your workload uses intermediate certificates, you must specify them in a `cert-chain.pem` file. 
+
+Add the certificates to {ProductShortName} by following these steps.
+
+1. Create a secret `cacert` including all the input files ca-cert.pem, ca-key.pem, root-cert.pem and cert-chain.pem:
++
+----
+$ oc create secret generic cacerts -n istio-system --from-file=samples/certs/ca-cert.pem \
+    --from-file=samples/certs/ca-key.pem --from-file=samples/certs/root-cert.pem \
+    --from-file=samples/certs/cert-chain.pem
+----
++
+2. Redeploy {ProductShortName} with `global.mtls.enabled` set to `true` and `security.selfSigned` set to `false`. {ProductShortName} reads the certificates and key from the secret-mount files.
++
+----
+$ istioctl manifest apply --set values.global.mtls.enabled=true,values.security.selfSigned=false
+----
++
+3. To make sure the workloads obtain the new certificates promptly, delete the secrets generated by {ProductShortName}, named `istio.\*`. In this example, `istio.default`. {ProductShortName} issues new certificates for the workloads.
+
+== Verifying your certificates
+
+Use the Bookinfo sample application to verify your certificates are mounted correctly.
+
+1. Retrieve the mounted certificates. In the following, we take the ratings pod as an example, and verify the certificates mounted on the pod. Set the pod name to RATINGSPOD:
```

Comment:
This is storing the pods name in the variable RATINGSPOD. It's not setting the pod name.